app.js

```diff
@@ -43,19 +70,26 @@ async function loadInstrument() {
   });
 }
 
+function mapButtons(availableButtons) {
```

Comment:
If `buttons` is set to a copy of `DEFAULT_BUTTONS`, then we defer the mapping to some future UI, and this function is no longer needed.